draft-ietf-ohai-ohttp.md

```diff
@@ -652,8 +652,10 @@ fields that are listed in the Connection header field.
 The oblivious proxy resource interacts with the oblivious request resource by
 constructing a request using the same restrictions as the client request, except
 that the target URI is the oblivious request resource.  The content of this
-request is copied from the client.  The oblivious proxy resource MUST NOT add
-information about the client to this request.
+request is copied from the client.  The oblivious proxy resource MUST NOT
```

Comment:
I'm personally in favour of changing the "MUST NOT add information about a client" to a SHOULD NOT, and craft verbiage to outline that this is a bad idea for fingerprinting/loss of anonymity etc reasons - basically considerations that a proxy should take into account (apologies @chris-wood I know that's what you originally had in your first commit, didn't get a chance to weigh in until now)

Reply:
No worries =)